beacon-chain/state/state-native/hasher.go

```diff
@@ -236,5 +238,30 @@ func ComputeFieldRootsWithHasher(ctx context.Context, state *BeaconState) ([][]b
 		fieldRoots[nativetypes.LatestExecutionPayloadHeader.RealPosition()] = executionPayloadRoot[:]
 	}
 
+	if state.version == version.Capella {
+		// Execution payload root.
+		executionPayloadRoot, err := state.latestExecutionPayloadHeaderCapella.HashTreeRoot()
+		if err != nil {
+			return nil, err
+		}
+		fieldRoots[nativetypes.LatestExecutionPayloadHeaderCapella.RealPosition()] = executionPayloadRoot[:]
+
+		// Withdrawal queue root.
+		withdrawalQueueRoot := []byte("stub")
```

Comment:
I left a gift for you @potuz

Reply:
These are hashed as normal SSZ lists of `Withdrawal` objects, doesn't  fastssz already cover this?

Reply:
discussed offline, we can reuse Nishant's vectorized merkleizer

Reply:
Do we intend to fix this ? Or is this meant to be in a follow up PR. i rather not have our hashing incomplete like this if we are adding in the whole state.